build.gradle.kts

```diff
@@ -28,12 +28,7 @@ buildscript {
 
     dependencies {
         classpath(libs.android.gradlePlugin)
-        classpath(libs.apollo.gradlePlugin)
-        classpath(libs.crashlytics.gradlePlugin)
-        classpath(libs.googleServices.gradlePlugin)
         classpath(libs.kotlin.gradlePlugin)
-        classpath(libs.licenses.gradlePlugin)
-        classpath(libs.lokalise.gradlePlugin)
```

Comment:
Aha so they don'tneed to be specified here whatsoever? Does it all happen automagically from the toml file?
Does it have something to do with the thing you added in buildSrc/serttings.gradle.kts or not?

Reply:
Right, the same way it would work if we didn't have them on the classpath but on the call site we defined them with the name AND the version. What this classpath was doing was having it not have to call the version right? but with the `alias` thing, it does give it the correct version as well, which is also consistent since it comes from the toml. So no need for the classpath solution.
I think I understand it now, thank you!